### PR TITLE
fix(avatar): Parse unicode characters in name for avatar component

### PR DESCRIPTION
**Pull request by legendik**
## Hey, I just made a Pull Request!

A small PR that fixes generating avatar from name / surname containing unicode characters (like mine :) ).

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/rare-crabs-taste.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': minor
+---
+
+Parse unicode characters in name for avatar component
```
Comment (benjdlambert) [inline .changeset/rare-crabs-taste.md:5]: Hmm, I think you might have to add the word `unicode` to `.github/vale/Vocab/Backstage/accept.txt`

Comment (benjdlambert) [inline .changeset/rare-crabs-taste.md:5]: Hmm, I think you might have to add the word `unicode` to `.github/vale/Vocab/Backstage/accept.txt`

**Comment by benjdlambert** [comment]
Can you mark the PR ready when it's ready for proper review + merge? :pray:

**Comment by legendik** [comment]
It is ready, thanks for review :)

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Thanks :tada:

**Pull request merged**